### PR TITLE
Add React cookie hooks

### PR DESCRIPTION
**Pull request by caprica-Six**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@
 - [CodeSandbox Starter Kit](https://codesandbox.io/s/7y6o4282lq)
 - [React Hooks Snippets for VS Code](https://marketplace.visualstudio.com/items?itemName=antmdvs.vscode-react-hooks-snippets)
 - [`hook-into-props`](https://github.com/juliettepretot/hook-into-props/tree/1e069a6c01c2a783100f2fea7709f56d8166a97e) Helper to build HOCs using hooks. Useful for using hooks with class components.
-- [`react-universal-hooks`](https://github.com/salvoravida/react-universal-hooks) React Universal Hooks: just use****** everywhere, Functional or Class Components
+- [`react-universal-hooks`](https://github.com/salvoravida/react-universal-hooks) React Universal Hooks: just use**\*\*** everywhere, Functional or Class Components
```
Comment (aarongarciah) [inline README.md:49]: Why this change?

Comment (caprica-Six) [reply README.md:49]: Umm looks like a weird Prettier onSave setting...

Comment (aarongarciah) [reply README.md:49]: Can you undo it?

Comment (caprica-Six) [reply README.md]: Sure, done, please check PR. Thanks

**Review by aarongarciah — APPROVED** [review]

**Comment by aarongarciah** [comment]
Thanks!
